ql/cashflows/overnightindexedcoupon.cpp

```diff
@@ -209,6 +213,27 @@ namespace QuantLib {
         }
     }
 
+    Real OvernightIndexedCoupon::accruedAmount(const Date& d) const {
+        if (d <= accrualStartDate_ || d > paymentDate_) {
+            // out of coupon range
+            return 0.0;
+        } else if (tradingExCoupon(d)) {
+            return -nominal() * rate(d) *
+                   dayCounter().yearFraction(d, std::max(d, accrualEndDate_), refPeriodStart_,
+                                             refPeriodEnd_);
```

Comment:
I'm not sure of the ex-coupon part, but let's keep it for the time being and think of the tests first.